README.md

```diff
@@ -46,6 +46,14 @@ For å installere node trenger man både *nodejs-legacy* og *npm*:
 sudo apt-get install nodejs-legacy npm git
 ```
 
+#### Mac-brukere merk dette! 
+For å kjøre gulp trenger du *pandoc*. Den enkleste måten å installere pandoc er
```

Comment:
Har [startet](https://github.com/arve0/codeclub_lesson_builder/issues/129) [halvveis](https://github.com/arve0/markdown-it-header-sections) med en overgang til [markdown-it](https://github.com/markdown-it/markdown-it), som er en markdown-parser skrevet i js. Problemer som dette vil da forsvinne, og det burde være relativt enkelt å lage en [live-editor](http://markdown-it.github.io/) for nettleseren.